packages/blocks/src/attachment-block/attachment-model.ts

```diff
@@ -21,6 +21,10 @@ export type AttachmentBlockProps = {
    */
   type: string;
   caption?: string;
+  // `loadingKey` was used to indicate whether the attachment is loading,
+  // which is currently unused but no breaking change is needed.
+  // The `loadingKey` and `sourceId` should not be existed at the same time.
+  // loadingKey?: string | null;
```

Comment:
I'd consider removing an unused model key as non breaking change, but we should list the definition of that field in the codebase comment, which could be good for potential debugging in the future.

cc @Flrande @Saul-Mirone @doouding @regischen @donteatfriedrice